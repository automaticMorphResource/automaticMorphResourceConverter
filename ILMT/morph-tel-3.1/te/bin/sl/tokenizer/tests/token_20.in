సింగపూర్ కంటే ఏడురెట్లు పెద్దదైన ఈ కంగారూ దీవి అద్భుతమైన వన్యప్రాణులకు, వృక్షజాతులకు పుట్టినిల్లు. 
