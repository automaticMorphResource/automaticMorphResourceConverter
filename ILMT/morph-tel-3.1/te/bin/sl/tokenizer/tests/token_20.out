<Sentence id="1">
1	సింగపూర్	unk
2	కంటే	unk
3	ఏడురెట్లు	unk
4	పెద్దదైన	unk
5	ఈ	unk
6	కంగారూ	unk
7	దీవి	unk
8	అద్భుతమైన	unk
9	వన్యప్రాణులకు	unk
10	,	unk
11	వృక్షజాతులకు	unk
12	పుట్టినిల్లు	unk
13	.	unk
</Sentence>

